express-app/controllers/estudio.js

```diff
@@ -0,0 +1,176 @@
+const Estudio = require('../models/estudio');
+const Familia = require('../models/familia');
+const familyController = require('./family');
+module.exports = {
+  /**
+  * This function shows the family form, IF the url query is empty
+  * then it means that the familia and estudio is created from scratch, IF NOT then
+  * it means it is a estudio that is going to be updated.
+  *
+  * @event
+  * @param {object} request - request object 
+  * @param {object} response - response object.
+  */
+  showFamilyForm: function(request, response) {
+    //retrieve estudio id from url
+    let estudioId = request.query.estudioId;
+    if (estudioId) {
+      Estudio.findOne({
+        _id: estudioId
+      })
+      .then((myEstudio) => {
+        response.render('family',  {
+          estudioId: myEstudio._id, 
+          family: myEstudio.familia
+        });
+      })
+      .catch((error) => {
+        //no estudio found
+        console.log(error);
+      })
+    }
+    else {
+      response.render('family');
```

Comment:
No, what this does is render the family view without information of a previously created Estudio.  At line 17 is an IF condition that checks if there is an ID at the query url.

Maybe I wasn't clear with the documentation of this function, do you need me to be more specific?

@erikiado